tools/create_entry_points.py

```diff
@@ -39,13 +42,23 @@
 def main():
   tools_dir = os.path.dirname(os.path.abspath(__file__))
   root_dir = os.path.dirname(tools_dir)
-  sh_file = os.path.join(tools_dir, 'run_python.sh')
-  bat_file = os.path.join(tools_dir, 'run_python.bat')
-  for entry_point in entry_points:
-    if os.path.exists(os.path.join(root_dir, entry_point)):
-        os.remove(os.path.join(root_dir, entry_point))
-    shutil.copy2(sh_file, os.path.join(root_dir, entry_point))
-    shutil.copy2(bat_file, os.path.join(root_dir, entry_point) + '.bat')
+
+  def generate_entry_points(cmd, path):
+    sh_file = path + '.sh'
+    bat_file = path + '.bat'
+    for entry_point in cmd:
+      dst = os.path.join(root_dir, entry_point)
+      if os.path.exists(dst):
+          os.remove(dst)
+      shutil.copy2(sh_file, dst)
+
+      dst += '.bat'
+      if os.path.exists(dst):
+          os.remove(dst)
+      shutil.copy2(bat_file, dst)
+
+  generate_entry_points(entry_points, os.path.join(tools_dir, 'run_python'))
+  generate_entry_points(compiler_entry_points, os.path.join(tools_dir, 'run_python_compiler'))
```

Comment:
I guess there are two new files here that need to be added?

Reply:
Thanks, indeed missed adding those.